spec/lib/exif_metadata_spec.rb

```diff
@@ -0,0 +1,77 @@
+require File.dirname(__FILE__) + '/../spec_helper.rb'
+
+describe ExifMetadata, 'extract' do
```

Comment:
Another style issue: we use double quotes pretty much across the board.

Reply:
Have you ever considered adding in a structured linter/style guide such as a `rubocop.yml`? Even if it wasn't wired up to run locally or in CI/CD or even make any changes, it could serve as reference for the style guide and be incorporated into many editors that support it and help all new code conform to the style specified.

Reply:
I'll think about it. We use eslint for JS and while it does encode our opinions about style, I also feel like I spend a lot of time making non-functional changes to enforce style compliance, so I'm not sure it's a net gain.

Reply:
Oh for sure. Definitely wouldn't want to start enforcing the guide until there's some major time set aside and even then it would probably need to be phased. More as an unenforced documentation of style guide than anything. Is it documented elsewhere that I may have missed?